moshi/src/test/java/com/squareup/moshi/ClassJsonAdapterTest.java

```diff
@@ -25,7 +25,6 @@
 import org.junit.Test;
 
 import static com.squareup.moshi.TestUtil.newReader;
-import static com.squareup.moshi.Util.NO_ANNOTATIONS;
```

Comment:
Revert this whole file's changes (or just static import so they match). History noise.